packages/now-build-utils/src/index.ts

```diff
@@ -111,9 +112,11 @@ export const getPlatformEnv = (name: string): string | undefined => {
   const n = process.env[nName];
   if (typeof v === 'string') {
     if (typeof n === 'string') {
-      throw new Error(
-        `Both "${vName}" and "${nName}" env vars are defined. Please only define the "${vName}" env var`
-      );
+      throw new NowBuildError({
+        code: 'CONFLICTING_ENV_VAR_NAMES',
+        message: `Both "${vName}" and "${nName}" env vars are defined. Please only define the "${vName}" env var.`,
+        link: 'https://vercel.link/combining-old-and-new-config',
```

Comment:
This link is 404

Reply:
Also, I would suggest changing `combining-old-and-new-config` to `multi-config` so its a little shorter

Reply:
PR is here: https://github.com/vercel/docs/pull/1899

The name was defined in the CH story, so @leo will need to make the final call on that.